app/__init__.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask
 
-app = Flask(__name__)
+app = Flask(__name__, static_folder="static")
```

Comment:
`static_folder` should got in a config file and default to `"static"` if it us unset.